### PR TITLE
svdsande Mozilla Firefox Goodie Cheat Sheet

### PR DESCRIPTION
**Pull request by svdsande**
What does your Instant Answer do?
This instant answer or cheat sheet provides a large list with all the shortcuts that are available for Mozilla Firefox today.

What problem does your Instant Answer solve (Why is it better than organic links)?
DuckDuckGo users can quickly see what shortcuts there are available for Mozilla Firefox, this can save time searching the internet.

What is the data source for your Instant Answer? (Provide a link if possible)
This is my data source: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly?redirectlocale=en-US&redirectslug=Keyboard+shortcuts

Why did you choose this data source?
Because this source is the official support page from Mozilla Firefox, so I know the information is correct. :)

Are there any other alternative (better) data sources?
- http://www-archive.mozilla.org/docs/end-user/moz_shortcuts.html
- http://www.accessfirefox.org/Firefox_Keyboard_and_Mouse_Shortcuts.html

What are some example queries that trigger this Instant Answer?
- firefox cheat sheet
- mozilla firefox cheat sheet

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
This instant answer is especially useful for all Mozilla Firefox user around the world.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
I posted my idea on the DuckDuckHack forum: https://duck.co/ideas/idea/6032/firefox-keyboard-shortcuts-cheat-sheet

Which existing Instant Answers will this one supercede/overlap with?
My instant answer will overlap with this one; https://duckduckgo.com/?q=awesome+bar+cheat+sheet&ia=cheatsheet

Are you having any problems? Do you need our help with anything?
No problems for me :) Last time I build a cheat sheet for Sublime Text but someone from the DuckDuckGo staff told me that someone else was building that cheat sheet. This time I checked if there was a Firefox shortcut cheat sheet.

Where did you hear about DuckDuckHack? (For first time contributors)

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
screenshot one
screenshot two

### CODE REVIEW

**Comment by MrChrisW** [comment]
@svdsande Thanks for contributing. Looks like there's already a Firefox CheatSheet PR (#1242) however it looks like yours includes additional shortcuts. :+1:

File: share/goodie/cheat_sheets/firefox.json [hunk 1]
```diff
@@ -0,0 +1,463 @@
+{
+    "id": "firefox_cheat_sheet", 
+
+    "name": "Mozilla Firefox",
+
+    "description": "Free and open-source web browser.", 
+    
+    "metadata": { 
+        "sourceName": "Mozilla Firefox",
+        "sourceUrl": "https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly?redirectlocale=en-US&redirectslug=Keyboard+shortcuts"
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/firefox.json]: Can this be changed to `https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly`

**Comment by moollaza** [comment]
@svdsande as @MrChrisW said, thanks a ton! We really do appreciate it :smile: 

Seeing as the other PR came first, I have to give precedence to it. Would you like to submit a new PR with improvements to that cheat sheet instead? It's just about to be merged...

**Comment by svdsande** [comment]
@moollaza @MrChrisW I thought I was the first with this cheat sheet idea. :( But I will make a new Pull Request with improvements to that cheat sheet. In The Netherlands where I live we have a saying "3 keer is scheepsrecht", this means that everything goes right the third time you try something. So the next time (my third attempt) I will do extra research for my cheat sheet idea so I know I will be the first :+1:

**Comment by MrChrisW** [comment]
@svdsande No problem! We have a similar saying "third time lucky" :smile:

You're always welcome to ping me through Gitter.im if you have any questions. There's always lots of other IA ideas you can look into on => https://duck.co/ideas

Looking forward to your next PR

**Comment by MrChrisW** [comment]
@svdsande P.S Keep your eye on #1242 and once merged you can PR your changes!

**Comment by moollaza** [comment]
@svdsande #1242 has now been merged!

**Pull request closed without merging**